blocks/components/editable/index.js

```diff
@@ -155,14 +156,15 @@ export default class Editable extends wp.element.Component {
 	}
 
 	render() {
-		const { tagName: Tag = 'div', style, className } = this.props;
-		const classes = classnames( 'blocks-editable', className );
+		const { tagName: Tag = 'div', style, className, value, placeholder } = this.props;
+		const classes = classnames( 'blocks-editable', className, { 'is-empty': ! value } );
```

Comment:
This and the above `onChange` logic on `value` will need some reconsideration if #466 is to come to be.